core/src/main/java/org/libreoffice/lots/db/SearchStrategy.java

```diff
@@ -32,35 +32,32 @@
 import org.libreoffice.lots.config.ConfigThingy;
 
 /**
- * Eine Suchstrategie liefert für eine gegebene Wortzahl eine Liste von Templates für
- * Suchanfragen, die der Reihe nach mit den Wörtern probiert werden sollen bis ein
- * Ergebnis gefunden ist.
+ * A search strategy provides a list of templates for search queries for a given number of words,
+ * which should be tried in sequence with the words until a result is found.
  */
 public class SearchStrategy
 {
   /**
-   * Bildet eine Wortanzahl ab auf eine Liste von {@link Query}-Objekten, die
-   * passende Templates darstellen.
+   * Maps a word count to a list of {@link Query} objects that represent matching templates.
    */
   private Map<Integer, List<Query>> mapWordcountToListOfQuerys;
 
   /**
-   * {@link #mapWordcountToListOfQuerys} wird per Referenz eingebunden und
-   * entsprechende Ergebnisse aus dieser Map werden von {@link #getTemplate(int)}
-   * zurückgeliefert.
+   * {@link #mapWordcountToListOfQuerys} is included by reference and
+   * corresponding results from this map are returned by {@link #getTemplate(int)}.
    */
+  
```

Comment:
There are several additional newlines in this pull request without any obvious pattern when they are added and when they are skipped...